ca/ca.go

```diff
@@ -362,6 +370,18 @@ func New(log *log.Logger, db *db.MemoryStore, ocspResponderURL string, alternate
 	for i := 0; i < len(ca.chains); i++ {
 		ca.chains[i] = ca.newChain(intermediateKey, intermediateSubject, subjectKeyID, chainLength)
 	}
+
+	if certificateLifetime != 0 {
+		if certificateLifetime > 0 {
+			ca.certLifetime = certificateLifetime
+			ca.log.Printf("Using user defined certificate lifetime of %d days", certificateLifetime)
+		} else {
+			ca.log.Printf("certificateLifetime (%d) is not a valid value (valid values: integer > 0), using default certificate lifetime of %d days", certificateLifetime, ca.certLifetime)
+		}
+	} else {
+		ca.log.Printf("Using default certificate lifetime of %d days", ca.certLifetime)
+	}
```

Comment:
Honestly I'd just simplify this to
```suggestion
	if certValidityPeriod != 0 {
		ca.certValidityPeriod = certValidityPeriod
	}
	ca.log.Printf("Using certificate validity period of %d seconds", ca.certValidityPeriod)
```
I don't think it's super important for the log message to indicate whether or not it's using the default value; it should just indicate what the value is.